modules/ROOT/pages/assembly_cli-reference.adoc

```diff
@@ -5,15 +5,15 @@
 
 :context: cli-reference
```

Comment:
Do we want to change the title to op-tkn-cli... or do you want to do that in a separate PR?

Reply:
I think we can do this as part of moving the assembly to OCP